ROADMAP.md

```diff
@@ -0,0 +1,71 @@
+# Roadmap
+
+> This document describes what additional tools/configs could be added to this monorepo.
+
+## New packages to add
+
+### ESLint
+
+We should be able to lint our javascript projects through ESLint and its configuration.
+
+#### Base
+
+Base configuration which should be used everywhere.
```

Comment:
Could you give an example?

Reply:
https://github.com/strvcom/code-quality-tools/tree/master/packages/eslint-config-base

Reply:
Oh, I'm sorry, I missed it's all subsections of `### ESLint`… My bad!